pkg/controller/uniteddeployment/uniteddeployment_update.go

```diff
@@ -132,6 +144,11 @@ func (r *ReconcileUnitedDeployment) manageSubsetProvision(ud *appsv1alpha1.Unite
 			return nil
 		})
 		if createdErr == nil {
+			// When a new subset is created, regardless of whether it contains newly created Pods,
```

Comment:
is it really necessary to trigger the requeue here given that manageUnschedulableStatusForExistingSubset had already push some duration time.

Reply:
manageUnscheddulableStatusForExistingSubset pushes some duration time only for existing subsets, new subsets is not processed